rdmd_test.d

```diff
@@ -675,11 +678,11 @@ void runFallbackTest(string rdmdApp, string buildCompiler, string model)
        if an explicit --compiler flag is not provided, rdmd should
        search its own binary path first when looking for the default
        compiler (determined by the compiler used to build it) */
-    string localDMD = buildPath(tempDir(), baseName(buildCompiler));
-    std.file.write(localDMD, "empty shell");
+    string localDMD = buildPath(tempDir(), baseName(buildCompiler).setExtension(binExt));
+    std.file.write(localDMD, ""); // An empty file avoids the "Not a valid 16-bit application" pop-up on Windows
     scope(exit) std.file.remove(localDMD);
 
     auto res = execute(rdmdApp ~ [modelSwitch(model), "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
-    assert(res.status == 1, res.output);
-    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
+    enforce(res.status == 1, res.output);
+    enforce(res.output.canFind(format(`spawn [%(%s%),`, localDMD.only)), localDMD ~ " would not have been executed");
```

Comment:
`was not executed` seems like a better description of the failure ... ?

Reply:
Right, I had the misconception that we were using `--dry-run` here.